packages/scss/src/components/readMore/component.scss

```diff
@@ -0,0 +1,54 @@
+@use '@lucca-front/scss/src/commons/utils/namespace';
+
+@use '@lucca-front/scss/src/commons/utils/a11y';
+
+@mixin component($atRoot: namespace.$defaultAtRoot) {
+	position: relative;
+	display: block;
+
+	@at-root ($atRoot) {
+		.readMore-link.link {
+			position: absolute;
+			inset: auto 0 var(--pr-t-spacings-50) auto;
+			display: var(--components-readMore-link-display);
+			flex-direction: column;
+			z-index: 1;
+			line-height: var(--sizes-M-lineHeight);
+			background-image: linear-gradient(
+				to right,
+				transparent,
+				var(--components-readMore-link-backgroudColor) calc(var(--components-readMore-link-paddingInlineStart) - var(--pr-t-spacings-100))
+			);
+			padding-inline-start: var(--components-readMore-link-paddingInlineStart);
+			font-weight: 600;
+			text-decoration: none;
+		}
+
+		.readMore-textFlow {
```

Comment:
The name sounds confusing: it feels like it's related to Text Flow component. What about `readMore-text` ou `readMore-content` ?

Reply:
As you've seen below, it's actually a textFlow that's used here. The components are nested, and the class allows you to see this without a doubt.